examples/Readme.md

```diff
@@ -27,6 +27,21 @@ yarn install
 yarn build
 ```
 
+For those who have problem when run start, you must install all the peer dependencies first on `package.json`
+
```

Comment:
To my knowledge, the general advice is to not use npm because it's not compatible with the yarn.lock file and we won't get the same versions of dependencies on install.

Reply:
Yes, i mean for the npm user, if he wants to try slate out then he must install yarn, and it is not a good experience.

Reply:
Yes, but providing instructions to an NPM user will imply that this tool can be correctly built with NPM. It cannot be built with NPM because it does not have the NPM lock file, it only has the yarn lock file. Therefore building with NPM is not guaranteed to pull down the same versions of each dependency.